First pass at C++ docs. (#671)
rive-app/rive-docs: runtimes/cpp/index.mdx

---
title: "C++ Runtime"
description: "Load, advance, and render Rive content from C++."
sidebarTitle: "Overview"
---

The Rive C++ runtime (`rive-cpp`) is the lowest-level Rive runtime. It loads
`.riv` files, advances state machines and animations, and draws into any
[Renderer](/runtimes/cpp/external-renderer) — including Rive's own GPU
[Renderer](/runtimes/cpp/renderers) (Metal, Vulkan, D3D11, D3D12, OpenGL/WebGL).

> Most other Rive runtimes (Apple, Android, Flutter, Unity, Unreal) wrap this

Reviewer: Most, not all?

Author: Changed

> library. Reach for it directly when you are:

Reviewer: ```suggestion
library. 

Use the C++ runtime when you are:
```

Author: Yeah, this is wonky grammar. Changed.
